src/lib/callback-trigger-point.ts

```diff
@@ -15,32 +15,48 @@
  * limitations under the License.
  */
 /**
- * @fileoverview A DrawTriggerPoint object maintains an x and y coordinate pair
- * and invokes the coordinator object's queueDraw() whenever either are set.
- * Used for the offset and scale properties.
+ * @fileoverview A CallbackTriggerPoint object maintains an x and y coordinate
+ * pair and invokes a provided callback whenever either are set. Used for the
+ * offset and scale properties.
  */
 
-export class DrawTriggerPoint {
+export class CallbackTriggerPoint {
```

Comment:
Renamed this class so that it could trigger a generic callback rather than a specific method on the coordinator. This facilitated keeping track of whether the view had been initialized (any changes to components of offset or scale).